constitution.txt

```diff
@@ -598,3 +598,25 @@ The financial year of the association is:
 
 Note: Schedule 1 of the Act provides that an association’s constitution is to address the association’s financial year.
 
+
+43. Objects of the Organisation
+
+(1) The objects for which the Organisation is established are:
+
+  (a) to provide  facilitate communication of information relating to inventions, results, techniques, and governance of open source;
+
+  (b) to support the creators of open source and open source projects;
```

Comment:
```suggestion
  (b) to support open source projects and the creators of open source ;
```
Re-ordered the noun phrases to be more clear that we're talking about projects and creators rather than two types of creator.